README.md

```diff
@@ -55,6 +55,13 @@ This directive is always required.
 Maximum HTTP body size this module can generate SXG from. Default value is
 `67108864` (64 MiB).
 
+#### sxg\_cert\_path
+
+The nginx will generate and serve CBOR encoded certificate file in this path.
+This is convenient when you want to serve certificate CBOR file from nginx.
```

Comment:
This is convenient when you want to serve **a** certificate CBOR file from nginx.

Reply:
a CBOR certificate file? And personally I'm not comfort to see "convenient" in description because I expect objective ones.

Reply:
removed.